aims/aim1.md

```diff
@@ -1,46 +1,48 @@
+We describe our standard approach here, as well as a couple selected techniques that we anticipate will be particularly helpful in this setting such as zero-inflated loss and data augmentation, but we expect to also employ other strategies where warranted.
+
+Our standard approach is to first perform an extensive grid search to identify VAE architectures and hyperparameters that are amenable to this context.
+Some parameter settings can be easily ruled out when models fail to train.
+Chris Probert noted on our proposal's GitHub repository [@url:https://github.com/greenelab/czi-rfa/issues/11] that numerous manuscripts demonstrated the advantages of zero-inflated loss in the setting of single cell sequencing data [@doi:10.1186/s13059-015-0805-z @arxiv:1610.05857 @doi:10.1186/s13059-017-1188-0].
+We will evaluate multiple types of reconstruction loss during training.
+These efforts will allow us to identify a subset of parameter combinations that are worth exploring with new approaches designed specifically for this type of genomic data.
+
+For single cell RNA-seq data, we also expect to be able to take advantage of data augmentation for the first time with genomic data.
+Data augmentation is widely used in image analysis, where the goal is to identify relevant features from a corpus of features that also have irrelevant differences.
+To understand data augmentation, imagine pathology slides that have been scanned.
+Each slide may be prepared and scanned in a subtly different orientation depending on who generated the data or at somewhat different magnifications depending on the center that performed the imaging.
+A deep learning method may learn to identify these subtle differences, which is undesirable.
+It's also possible that there are simply too few slides to train a deep learning algorithm.
+To address these challenges, deep learning practitioners can apply arbitrary rotations, zooms, and other irrelevant transformations to image data during training.
+This process is called data augmentation.
+
+No data augmentation approaches for genomic data have been published to date.
+However, we expect that very fast abundance estimates [@doi:10.1038/nmeth.4197 @10.1038/nbt.3519] will make a data augmentation approach feasible in this domain as well.
+Resampling reads to generate abundance estimates can help to capture the uncertainty in the data, akin to arbitrary rotations.
+Subsampling reads to generate abundance estimates can help to capture changes related to sequencing depth but unrelated to the biology, akin to arbitrary zooms.
+Therefore, we plan to collaborate with Rob Patro's laboratory via our collaborative network to implement approaches including but not limited to rapid subsampling and bootstrapping to generate augmented training data.
+We posit that genomic data augmentation will improve latent feature generalization by isolating cell-type as opposed to technical features and increasing the effective sample size during training.
```

Comment:
Not really sure what's technically correct here. I think of it as cell-type features, which would have the hyphen. Going to reword to avoid the decision :smile: